laws/src/main/scala/cats/laws/InvariandAddMonoidalLaws.scala

```diff
@@ -0,0 +1,30 @@
+package cats
+package laws
+
+import cats.data.INothing
+
+/**
+ * Laws that must be obeyed by any `cats.InvariantChoosable`.
+ */
+trait InvariantChoosableLaws[F[_]] extends InvariantChoiceLaws[F] {
```

Comment:
The file is called `InvariandAddMonoidalLaws`. It could be `InvariantChoosableLaws`